autoload/direnv.vim

```diff
@@ -0,0 +1,83 @@
+" direnv.vim - support for direnv <http://direnv.net>
+" Author:       zimbatm <http://zimbatm.com/> & Hauleth <lukasz@niemier.pl>
+" Version:      0.3
+
+scriptencoding utf-8
+
+let s:direnv_cmd = get(g:, 'direnv_cmd', 'direnv')
+let s:direnv_auto = get(g:, 'direnv_auto', 1)
+let s:job_status = { 'running': 0, 'stdout': [''], 'stderr': [''] }
+
+function! direnv#auto() abort
```

Comment:
Moving this function to `autoload` makes no sense as it is launched always when sourcing `plugin/direnv.vim`.

Reply:
I only moved this because of line 8 (the moving of `let s:direnv_auto`); it’s probably not necessary to move.